server/controllers/locationDetailsController.test.ts

```diff
@@ -1,27 +1,26 @@
 import { PrisonerMockDataA, PrisonerMockDataB } from '../data/localMockData/prisoner'
 import { inmateDetailMock } from '../data/localMockData/inmateDetailMock'
 import { CaseLoadsDummyDataA } from '../data/localMockData/caseLoad'
-import PrisonerLocationDetailsController from './prisonerLocationDetailsController'
+import LocationDetailsController from './locationDetailsController'
```

Comment:
A bit more renaming going on here to remove 'prisoner' in the name since that's implicit given the service is all about the prisoner.